openml/testing.py

```diff
@@ -252,6 +253,58 @@ def _check_fold_timing_evaluations(
                         self.assertLessEqual(evaluation, max_val)
 
 
+def check_task_existence(
+    task_type: TaskType, dataset_id: int, target_name: str, **kwargs
+) -> Union[int, None]:
+    """Checks if any task with exists on test server that matches the meta data.
+
+    Parameter
+    ---------
+    task_type : openml.tasks.TaskType
+        ID of the task type as detailed `here <https://www.openml.org/search?type=task_type>`_.
+        - Supervised classification: 1
```

Comment:
You can drop this enumeration as `task_type` is an ENUM now and all legal values are described in there.